server/routes/participant.js

```diff
@@ -453,7 +458,15 @@ employerActionsRouter.post(
       participant_id: participantId,
       status,
     });
-    return res.status(201).json({ data: result });
+    let returnStatus = 201;
+    if (
```

Comment:
While I don't love this, it seems the best way to properly return an error.

I've seen other areas of the backend return `success: true` from the service function. The problem with that is that it would require a fair bit more refactoring, since `setParticipantStatus` is called by a bunch of other places.